Code/Tools/ProjectManager/Source/ProjectExportController.cpp

```diff
@@ -122,16 +122,17 @@ namespace O3DE::ProjectManager
                 if (logFilePathQuery.IsSuccess())
                 {
                     QMessageBox::critical(m_parent,
-                                      tr("%1\nYou can check the logs in the following directory:\n%2")
+                                      QString("%1\nYou can check the logs in the following directory:\n%2")
```

Comment:
Why is this a `QString` now instead of a [tr](https://doc.qt.io/qt-6/qobject.html#tr)?  `tr` returns a translated QString anyways.